glean-core/ffi/src/lib.rs

```diff
@@ -101,6 +101,7 @@ pub struct FfiConfiguration<'a> {
     package_name: FfiStr<'a>,
     upload_enabled: u8,
     max_events: Option<&'a i32>,
+    delay_ping_lifetime_io: u8,
```

Comment:
Do we actually need to expose this over FFI?
The platform layers don't expose it to users right now, just setting default values.

The only intended usage right now is Firefox and for that we plan to use the Rust API directly.
IMO it makes more sense to keep it out of the FFI for now and add it when necessary.
Wdyt, @Dexterp37?

Reply:
Also @chutten 

Only minor thing: for the API preview I want to land tomorrow we will use the FFI. Can we live without the delay initially? The newer API i propose would not go through FFI.

Reply:
@badboy mh, I feel like exposing the option across all the platforms for consistency would be better/make it easier for understanding and documenting what's available where. However I have no strong opinion on this.

With respect to the FOGotype, no, we don't need to block on it. This is for project FOG, not for the fogotype :)

Reply:
All right, then we keep it as is (and I'm personally gonna remove it from the FFI layer in 3 months if we never use it :P)